src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -405,15 +376,50 @@ tests:
           set to G1 and the SceneID field set to 0x01."
       PICS: S.S.C01.Rsp && PICS_SKIP_SAMPLE_APP
       verification: |
```

Comment:
My understanding is TC-S-2.3 is a server test. Why are you relying on manual verification when you can send the `ViewScene` and validate that response using yaml test infrastructure?

Reply:
Because depending on the device tested, the clusters present in the extension field sets will vary so we cannot predict the content and value of a ViewScene. 
E.g. The scene added in the Step 2 will not be the same if we are testing an on-off light, a dimmable light or a color light.
Short term solution is to have those steps manual, long term would be to provide a way to device makers to provide their clusters/attributes.